<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ForceSim</string>
  </property>
  <property name="windowOpacity">
   <double>0.500000000000000</double>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>107</y>
     <width>521</width>
     <height>521</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>14</y>
     <width>521</width>
     <height>93</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_strength">
      <property name="text">
       <string>中心力度     </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_strength">
      <property name="minimum">
       <double>5.000000000000000</double>
      </property>
      <property name="maximum">
       <double>500.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>5.000000000000000</double>
      </property>
      <property name="value">
       <double>80.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_row_num">
      <property name="text">
       <string>行密度</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="spinBox_row_num">
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QCheckBox" name="checkBox_reject">
      <property name="text">
       <string>边界穿越/反弹</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_ratio">
      <property name="text">
       <string>阻力系数</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_ratio">
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>5.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_col_num">
      <property name="text">
       <string>列密度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="spinBox_col_num">
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_background_color">
      <property name="text">
       <string>背景色</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_choose_color">
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_node_color">
      <property name="text">
       <string>材质色</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_node_color">
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_init">
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/icons/icons/play.png);}
QPushButton:hover{border-image: url(:/new/icons/icons/play-hover.png);}
QPushButton:pressed{border-image: url(:/window/pic/ui/button_pushed.png);}</string>
      </property>
      <property name="text">
       <string>初始化</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>随机材质颜色</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
